website/docs/r/control_mute.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: "turbot"
+title: "turbot"
+template: Documentation
+page_title: "Turbot: turbot_control_mute"
+nav:
+  title: turbot_control_mute
+---
+
+# turbot\_control\_mute
+
+`Turbot Control Mute` allows to mute a control on a given resource.
```

Comment:
I think we can add a verbose description here? This doc might be helpful. https://turbot.com/guardrails/docs/concepts/controls#muting-controls